.github/workflows/build.yml

```diff
@@ -0,0 +1,152 @@
+name: "Build"
+
+on:
+  push:
+  pull_request:
+
+jobs:
+
+  test:
+    name: Maven Test & Package
+    runs-on: ubuntu-latest
+
+    env:
+      JAVA_DISTRIBUTION: microsoft
```

Comment:
Any reason for the micosoft distribution vs [temurin](https://github.com/actions/setup-java/blob/main/README.md#Supported-distributions)?